js/functions.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const querystring = require('querystring');
+
+module.exports = (config = {}) => {
+
+  config = Object.assign({
+    add_version: true,
+  }, config);
+
+  return {
+    asset: (asset_path) => {
+      if (!config.path) {
```

Comment:
1. This should also be `asset_path_document_root` or sans module name prefix just `path_document_root` if the config is scoped to the asset module.

1. Can you add installation / usage examples for both PHP and JS to the README.txt?

Reply:
Done, please review and edit if necessary 🙂